action/messageHubFeed.js

```diff
@@ -55,6 +62,35 @@ function main(params) {
     }
 }
 
+function checkMessageHubCredentials(params) {
+    // listing topics seems to be the simplest way to check auth
+    var topicURL = params.kafka_admin_url + '/admin/topics';
```

Comment:
This is a new parameter that needs to be supplied when creating the trigger. It is part of the Bluemix credentials and will be "magically" present when using `package refresh`.